11-Backbonejs/collections.md

```diff
@@ -0,0 +1,164 @@
+# Organizing Data with Models and Collections
+
+As we've worked with JavaScript, up until this point all our data in has been stored as arrays of objects. This is fine for simple programs, but for more complex applications we'll need a more robust way to organize things. In the Backbone framework, this problem is solved through Models and Collections.
+
+## Learning Goals
+
+By the end of this lesson, you should be able to...
+
+- Explain what a Backbone collection is
+- Define and instantiate a Backbone Collection
+- Add models to a collection
+- Retrieve models from a collection
+- Iterate through models in a collection
+
+## What are Collections in Backbone?
+
+Collections are a specific type of model that keep track of a set of related objects, rather than one individual piece of data. The relationship between Collections and Models is like the one between arrays and objects.
+
+Thinking back to Rails and ActiveRecord, a Backbone Collection contains things that would be class methods for an AR Model: built-in methods similar to `.all`, `.find` and `.order`, as well as any custom functions that operate on a group of objects.
+
+## BookList
+
+Now that we've defined our `Book` Model, it's time to build a Collection to contain it. We will call this Collection `BookList`.
+
+### Defining The Collection
+
+1. Create a file `src/collections/book_list.js`
+1. Open that file in Atom, and add the following code:
+    ```javascript
+    import Backbone from 'backbone';
+    import Book from '../models/book';
+
+    const BookList = Backbone.Collection.extend({
```

Comment:
I did not know this was true. Do you have a source on this? I can't find anything about it in the Backbone docs, and people on the internet seem to do all sorts of things.

Reply:
These two refs:
https://stackoverflow.com/questions/8695254/what-are-some-backbone-js-naming-conventions
http://codebeerstartups.com/2012/12/8-collections-in-backbone-js-learning-backbone-js/

Reply:
If I have time to address this, it will be as a separate PR / change to the livecode. However, propagating it through all those branches will be a lot of work potentially.